VirtoCommerce.DynamicExpressionsModule.Web/Scripts/dynamicExpression.js

```diff
@@ -142,6 +142,12 @@ angular.module(moduleName, [])
           id: 'RewardItemGetOfRel',
           displayName: 'Get [] % off'
       });
+
```

Comment:
You should override already existing expression (the new expression type is not required)
And please check all others the relative amount based rewards elements and expressions to add this new field